moto/sns/responses.py

```diff
@@ -12,10 +12,6 @@ class SNSResponse(BaseResponse):
     def backend(self):
         return sns_backends[self.region]
 
-    @property
-    def request_json(self):
-        return 'JSON' in self.querystring.get('ContentType', [])
```

Comment:
Yes, I [added it to the base](https://github.com/spulec/moto/pull/550/files#diff-427c67d098e415aae0a327e597f8c7c7R258).